RELEASE_NOTES.md

```diff
@@ -1,5 +1,339 @@
+  # code to remove it that doesn't need a password
+end
+```
+
+### Removal of Legacy metadata.rb depends Version Constraints
+
+Support for the `<<` and `>>` version constraints in metadata.rb has been removed. This was an undocumented feature from the Chef 0.10 era, which which is not used in any cookbooks on the Supermarket. We are mentioning it since it is technically a breaking change, but it unlikely that this change will be impacting.
+
+Examples:
+
+```ruby
+depends 'windows', '<< 1.0'
+depends 'windows', '>> 1.0'
+```
+
+### Changes to Knife
+
+#### knife status --long uses cloud attribute
+
+The `knife status --long` resource now uses Ohai's cloud data instead of ec2 specific data. This improves, but changes, the data output for users on non-AWS clouds.
+
+#### knife-acl is now built-in
+
+The knife-acl gem is now part of Chef Infra Client, but many of the commands have been moved into the existing knife user and group namespaces.
+
+TODO: add details on the command changes from https://github.com/chef/chef/pull/9292
+
+#### knife download role/environment format update
+
+The `knife download role` and `knife download environment` commands now include all possible data fields including those without any data set. This new output behavior matches the behavior of other commands such as `knife role show` or `knife environment show`
+
+#### Deprecated knife cookbook site command removed
+
+The previously deprecated `knife cookbook site` commands have been removed. Use the `knife supermarket` commands instead.
+
+#### Deprecated knife data bag create -s short option removed
+
+The deprecated `knife data bag create -s` option that was not properly honored has been removed. Use the `--secret` option instead to set a data bag secret file during data bag creation.
+
+#### sites-cookbooks directory no longer in cookbook_path
+
+The legacy `sites-cookbooks` directory is no longer added the default `cookbook_path` value. With this change any users with a legacy `sites-cookbooks` directory will need to use the `-O` flag to override the cookbook directory when running commands such as `knife cookbook upload`.
+
+If you have a repository that contains a `site-cookbooks` directory we highly recommend using Policyfiles or Berkshelf to properly resolve these external cookbook dependencies without the need to copy them locally. Alternatively you can move the contents of this folder into your main cookbook directory and they will continue to be seen by knife commands.
+
+## New Resources
+
+### alternatives
+
+Thank @vkhatri
+
+### chef_client_cron
+
+### chef_client_systemd_timer
+
+### chef_client_windows_task
+
+### plist
+
+Thank Microsoft and @americanhanko
+
+### user_ulimit
+
+### windows_security_policy
+
+### windows_user_privilege
+
+## Improved Resources
+
+### compile_time on all resources
+
+The `compile_time` property is now available for all resource so that they can be set to run at compile time without the need forcing the action.
```

Comment:
Should this be "without forcing the action" or "without needing to force the action"?

```suggestion
The `compile_time` property is now available for all resources so that they can be set to run at compile time without the need forcing the action.
```